src/components/FormInput/FormInput.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { spacingSmall, colorRed } from '../../styles/designTokens';
+import Input from '../Input';
+
+const Label = styled.label`
+  margin-bottom: ${spacingSmall};
+`;
+
+const ErrorLabel = styled.div`
+  font-style: italic;
+  color: ${colorRed};
+`;
+
+const InputForm = ({ className, label, error, ...props }) => {
+  return (
+    <>
+      {label ? (
```

Comment:
Due to accessibility requirements form label should not be optional.

Reply:
Ok, I will make it required and remove the ternary validation.

Reply:
👍